OpenSim/Simulation/Model/Model.cpp

```diff
@@ -489,6 +494,46 @@ void Model::createMultibodySystem()
     addToSystem(*_system);
 }
 
+
+void Model::reorderCoordinatesAccordingToSystemMobilities()
```

Comment:
Why does this function rely on Topology? Is it because you need a State? Could you document this at the declaration of `Model::reorderCoordinatesAccordingToSystemMobilities()`?